ganga/GangaCore/GPIDev/Adapters/IVirtualization.py

```diff
@@ -17,7 +18,12 @@ def __init__(self, image):
         self.image = image
 
     _schema = Schema(Version(1, 0), {
-        'image': SimpleItem(defvalue="", typelist=[str], doc='Link to the container image')
+        'image': SimpleItem(defvalue="", typelist=[str], doc='Link to the container image'),
+        'username': SimpleItem(defvalue="", typelist=[str], doc='Deploy token username'),
+        'password': SimpleItem(defvalue="", typelist=[str], doc='Deploy token password'),
```

Comment:
Is it ok to keep a password like this?

Reply:
Fair question. It is what is called a deploy token which means that it is only valid for pulling the image and nothing else.